testing/integration-tests/src/light_client/mod.rs

```diff
@@ -34,54 +34,93 @@ use subxt_metadata::Metadata;
 
 type Client = OnlineClient<PolkadotConfig>;
 
+/// The Polkadot chainspec.
+const POLKADOT_SPEC: &str = include_str!("../../../../artifacts/demo_chain_specs/polkadot.json");
+
+/// Use the unstable backend for testing.
+const USE_UNSTABLE_BACKEND: bool = true;
```

Comment:
I guess it makes sense to test the light client with just the "unstable" backend, though we could have that light client test be a fn that takes in this bool and then run it twice to test both backends until the unstable one is stabilised? (and then remove this const)